src/widgets/RRule.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import Container from './Container';
+import { Input, RadioGroup, FormControlLabel, Radio, FormLabel } from '@material-ui/core';
+import RRule, { RRuleSet, Frequency } from 'rrule';
+
+interface PropsType { sendRRule: (rRule: string) => void }
+
+export default function RRuleEteSync(props: PropsType) {
+  const [frequency, setFrequency] = React.useState(RRuleSet.HOURLY);
+  const [interval, setInterval] = React.useState<number | undefined>();
+  const [ends, setEnds] = React.useState('never');
+  const [until, setUntil] = React.useState<Date | null>(null);
+  const [count, setCount] = React.useState<number | null | undefined>();
+  const rule = new RRule({
```

Comment:
No need to actually create an rrule object here. You can just have an object that looks like what you're passing to the RRule constructor as the main rule object.